src/pykeen/models/unimodal/complex.py

```diff
@@ -75,6 +75,8 @@ def __init__(
         preferred_device: DeviceHint = None,
         random_seed: Optional[int] = None,
         regularizer: Optional[Regularizer] = None,
+        entity_initializer=nn.init.normal_,
```

Comment:
does this break the CLI

Reply:
No it does not

Reply:
`TestComplex` says no :wink:

Reply:
https://github.com/pykeen/pykeen/pull/199/checks?check_run_id=1511414327#step:5:160

`TestExperimentIntegrity` does not like it though :-/

Reply:
good thing we have tests!